File: vs/Listas/Lista 1/div.c
#include <stdio.h>
#include <locale.h>

int main()
{
	setlocale(LC_ALL, "Portuguese"); // Declaração de idioma usado, para utilização de acentos.
	system("cls");
	
	float x,y,div;
	
	printf("Insira um valor para o dividendo:  ");
	scanf("%f",&x);
	
	printf("\nInsira um valor para o divisor:  ");
	scanf("%f",&y);
	
	if (y==0){
		printf("\nO divisor informado é igual a 0, não é possível dividir por 0.\nInsira outros valores\n\n");
		system("pause");  
		return main();
	}
	else {
		div= x/y;
		printf("\nO quociente da divisão %.2f/%.2f é igual a %.2f\n",x, y, div);
	}		
	return 0;	
}

File: vs/Listas/Lista 1/div.py
def calc():
    sn=0
    div=0
    x=float(input("\nInsira um valor para o dividendo:  "))
    y=float(input("\nInsira um valor para o divisor:  "))

    if y==0:
        sn= input("\nNão é possível dividir por 0.\n\nDeseja inserir novos números?[S/N]")
        if sn == 'S' or sn == 's':
            calc()
        else :
            print("\nAté mais!\n")
    else:
        div=x/y
        print("\nO valor da divisão é igual a: ")
        print(div)
calc()
